installers/go-agent/release/go-agent.init

```diff
@@ -96,7 +95,7 @@ stop_go_agent() {
     check_proc
 
     if [ $? -eq 0 ]; then
-        killproc -p $PID_FILE /usr/share/${SERVICE_NAME}/agent.sh >/dev/null
+        killproc /usr/share/${SERVICE_NAME}/stop-agent.sh ${SERVICE_NAME} service_mode
```

Comment:
My vote would be to just explicitly invoke `stop-agent.sh` and remove `killproc()` altogether since it's now a useless use of `eval`.

Suggesting to change this line to:

```
>/dev/null /usr/share/${SERVICE_NAME}/stop-agent.sh ${SERVICE_NAME} service_mode
```

Reply:
@ibnc No, I'm not sure why. @marques-work's suggestion makes sense to me. I wouldn't do `>/dev/null` though. My preference would be to leave it be a little verbose, especially if it fails to kill. If, on failure to kill, it shows a message on stderr, then `>/dev/null` should be fine.

Reply:
@arvindsv better to be paranoid and remove the `>/dev/null` :)